### ISSUE TITLE
Suggestion: How to get custom armors working

### ISSUE THREAD

**Issue by raponoves** [Yesssssman/epicfightmod#21]
I asked around in the "Mo'bends" Discord for how he got Custom 3D armor working, and this is the response I received

**"Question, I noticed with more recent versions you've been able to render 3D modeled armor onto the animated player skeleton. How did you do that?"**

_"Mo Bends actually mutated the player model in 1.0.0 instead of overriding it"_

_"It means that 0.24 replaced the player model with a custom one from Mo Bends, 1.0.0 uses a mutation system, so it takes the existing model cutting it up into smaller parts where needed and assigning new animations. This is also extended to the armor, if modders implement their armors correctly Mo Bends can then simply cut the armors along the same lines and move it around as needed"__

**"So basically instead of making a new skeleton entirely, they chopped up the old skeleton and added new parts where needed?"**

__"That's at least the basic gist of it"_

_"Pretty art to visualize it"__

![unknown](https://user-images.githubusercontent.com/53382034/108913543-1c2d4480-75f8-11eb-8ad6-ac2f2998b527.png)

So it sounds like the best way to make 3D armor work, would be to use the pre-existing Minecraft skeleton as a base, and modifying it to work like your animated skeleton, as opposed to the creation of a new one.

**Comment by AlexandreKayle** [comment]
Very nice ! I hope the creator sees this and really does, this mod is so good, however, being incompatible is sooo bad, because most will use it in an epic battle modpack, but the armor willn’t work properly, it’s a pity, hopefully that this method works

**Comment by Yesssssman** [comment]
I think I need some research on this. I'll let you know if there is some progression.

**Comment by AlexandreKayle** [comment]
Yeeess !! We did it

**Comment by raponoves** [comment]
All of his code is visible on Github I believe 

https://github.com/mobends/MoBends

Hopefully that is helpful.

**Comment by Yesssssman** [comment]
Ok, I succeed in importing a custom armor model to an animated model format. The next important thing is applying the animated position to each vertex. I'll post later when I succeed. Keep your eye on this!

![2021-03-07_14 49 31](https://user-images.githubusercontent.com/79469058/110230447-64cbf280-7f54-11eb-8572-0611b95bd751.png)
![2021-03-07_14 49 36](https://user-images.githubusercontent.com/79469058/110230455-6dbcc400-7f54-11eb-8365-17455df32ea1.png)
![2021-03-07_14 49 42](https://user-images.githubusercontent.com/79469058/110230457-6f868780-7f54-11eb-82a2-cf0834151708.png)
